### PR TITLE
Add sqlite make-based script

### PR DESCRIPTION
**Pull request by andreasstamos**
@StefanJum Add do.sh script for SQLite (https://github.com/unikraft-upb/scripts/issues/10) Team ByteBusters

### CODE REVIEW

File: make-based/app-sqlite/do.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+app_basename="sqlite"
+app_libs="musl sqlite"
+use_kvm="1"
+use_9p_rootfs="1"
+rootfs_9p="$PWD/sql_files"
```
Comment (StefanJum) [inline make-based/app-sqlite/do.sh:7]:
```suggestion
rootfs_9p="$PWD/files/sql_files"
```
I would create a new directory `files/sql_files` and add a dummy file so the user can test it. You can add the [`script.sql` script here](https://unikraft.org/docs/usage/make_build/#app-sqlite-for-x86_64).

**Review by StefanJum — CHANGES_REQUESTED** [review]
Hi @andreasstamos, thank you for this and sorry for the delay.

Besides the comments, can you please add the `Signed-off-by: Your Name <youremail@...> `line at the end of the commit message, and change the commit message to something like make-based: Add app-sqlite (similar to [this one](https://github.com/unikraft-upb/scripts/commit/fe42d87f0ccbfd92f43e6344eda39385c322f97c)).

**Comment by razvand** [comment]
@StefanJum , I forced push to this PR. See if it's OK.

**Review by StefanJum — APPROVED** [review]
All good, thanks.
Reviewed-by: Stefan Jumarea <stefanjumarea02@gmail.com>
Approved-by: Stefan Jumarea <stefanjumarea02@gmail.com>

**Pull request merged**